frontend/src/metabase/query_builder/components/DatasetEditor/DatasetFieldMetadataSidebar/DatasetFieldMetadataFkTargetPicker.tsx

```diff
@@ -18,6 +19,7 @@ export const DatasetFieldMetadataFkTargetPicker = ({
   const [formField, _meta, { setValue }] = useField("fk_target_field_id");
   const { data: idFields = [] } = useListDatabaseIdFieldsQuery({
     id: databaseId,
+    ...PLUGIN_FEATURE_LEVEL_PERMISSIONS.dataModelQueryProps,
```

Comment:
I'm not sure if this is needed here in models

Reply:
It seems right to me, but you might double check with someone who knows better.

Reply:
I double-checked this and it's not how it works in stats. Reverted: ce4ded9c3807b5f69ef63016eefa5969017a43